<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gauss</class>
 <widget class="QWidget" name="Gauss">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labLabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Метод Жордана-Гаусса</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <widget class="QScrollArea" name="srcArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="srcContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>556</width>
          <height>388</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="srcLayout">
           <item>
            <layout class="QHBoxLayout" name="conditionLayout">
             <item>
              <widget class="QTableWidget" name="conditionTableWidget">
               <property name="minimumSize">
                <size>
                 <width>182</width>
                 <height>182</height>
                </size>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <property name="rowCount">
                <number>3</number>
               </property>
               <property name="columnCount">
                <number>3</number>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>60</number>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>60</number>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderMinimumSectionSize">
                <number>23</number>
               </attribute>
               <row/>
               <row/>
               <row/>
               <column/>
               <column/>
               <column/>
               <item row="0" column="0">
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item row="0" column="1">
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item row="0" column="2">
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item row="1" column="0">
                <property name="text">
                 <string>4</string>
                </property>
               </item>
               <item row="1" column="1">
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item row="1" column="2">
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item row="2" column="0">
                <property name="text">
                 <string>9</string>
                </property>
               </item>
               <item row="2" column="1">
                <property name="text">
                 <string>3</string>
                </property>
               </item>
               <item row="2" column="2">
                <property name="text">
                 <string>1</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="conditionColumnWidget">
               <property name="minimumSize">
                <size>
                 <width>62</width>
                 <height>62</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>62</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAsNeeded</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAsNeeded</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustIgnored</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <property name="rowCount">
                <number>3</number>
               </property>
               <property name="columnCount">
                <number>1</number>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>60</number>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>60</number>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderMinimumSectionSize">
                <number>23</number>
               </attribute>
               <row/>
               <row/>
               <row/>
               <column/>
               <item row="0" column="0">
                <property name="text">
                 <string>0</string>
                </property>
               </item>
               <item row="1" column="0">
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item row="2" column="0">
                <property name="text">
                 <string>3</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="settingsLayout">
       <item>
        <widget class="QLabel" name="matrixSizeLabel">
         <property name="text">
          <string>Кол-во уравнений:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="matrixSizeSpin">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calculateButton">
         <property name="text">
          <string>Рассчитать</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="settingsSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
